src/openrct2/platform/macos.m

```diff
@@ -215,4 +215,15 @@ void platform_get_changelog_path(utf8 *outPath, size_t outSize)
     safe_strcat_path(outPath, "changelog.txt", outSize);
 }
 
+bool platform_get_steam_path(utf8 * outPath, size_t outSize)
+{
+    const char * steamPath = "~/Library/Application Support/Steam/steamapps/common";
```

Comment:
Like Linux, this won't work.
@marijnvdwerf or @LRFLEW might be able to tell how to retrieve the home folder.

Reply:
The same code for retrieving the home directory can be used for Linux and macOS:
https://github.com/OpenRCT2/OpenRCT2/blob/c4538496c536d7c21fa99d683f070439771eda03/src/openrct2/platform/posix.c#L710